samples/bluetooth/hci_usb/README.rst

```diff
@@ -21,3 +21,11 @@ This sample can be found under :zephyr_file:`samples/bluetooth/hci_usb` in the
 Zephyr tree.
 
 See :ref:`bluetooth samples section <bluetooth-samples>` for details.
+
+.. note::
```

Comment:
This note would apply to all bluetooth samples, not just this one.
This note might be better placed here:
https://docs.zephyrproject.org/latest/samples/bluetooth/bluetooth.html

Reply:
Right, I was considering this, so I'll change the location of the note. I decided to add it here at first as this HCI USB sample seems quite popular and as it only contains the controller it might be especially confusing that 2 images have to be built for nRF53.

Reply:
Updated